jenkins/Jenkinsfile-blossom.premerge-databricks

```diff
@@ -63,7 +65,7 @@ pipeline {
         DATABRICKS_HOST = DbUtils.getHost("$DB_TYPE")
         DATABRICKS_TOKEN = credentials("${DbUtils.getToken("$DB_TYPE")}")
         DATABRICKS_PUBKEY = credentials("SPARK_DATABRICKS_PUBKEY")
-        DATABRICKS_DRIVER = DbUtils.getDriver("$DB_TYPE")
+        DATABRICKS_DRIVER = 'Standard_NV36ads_A10_v5'
```

Comment:
Curious why this was changed to be hardcoded instead of parameterized?  Since this is a single-node cluster instance, I expected this to use the worker type as the driver type.  Would that not work here?